README.md

```diff
@@ -89,6 +89,7 @@ This repository is designed to help newbies practice using Git and GitHub by cre
 - [John Doe](./john-doe/README.md)
 - [Sebastian Harvey](./jsstar/README.md)
 - [White Collin](./whitecollin/)
+- [naidendev](./naidendev/README.md)
```

Comment:
put your name at the end of contributors

Reply:
fixed it